src/Dialog/Dialog.Component.js

```diff
@@ -0,0 +1,63 @@
+import path from 'path';
+import { Button, Dialog } from '..';
+import { ComponentPage, Example } from '../_playground';
+import React, { Component } from 'react';
+
+export class DialogComponent extends Component {
```

Comment:
Could this be a functional component?

Reply:
oh this is copied from the previous ModalComponent - it just looks new from the name change. I don't really want to rework all the docs 😆